### PR TITLE
User Handlers

### PR DESCRIPTION
**Pull request by CrispyCl**
### User
  
  - [X] Start command
  - [x] Help command
  - [x] Change language
  - [x] Balance info
  - [x] Daily token bonus
  
  ### Staff
  
  - [x] Start command
  - [x] Help command
  - [x] Change language
  
  ### Admin
  
  - [x] All staff actions
  - [x] Add staff
  - [x] Delete staff
  
  ### Other
  
  - [x] Translate it all
  - [x] Emoji in buttons

### CODE REVIEW

File: bot/handlers/admin.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+@router.message(CommandStart(), IsSuperAdminFilter())
+async def proccess_super_admin_start(message: Message) -> None:
+    button1 = KeyboardButton(text=_("Add admin"))
+    button2 = KeyboardButton(text=_("Add video"))
+    reply_kb: ReplyKeyboardMarkup = ReplyKeyboardMarkup(keyboard=[[button1, button2]], resize_keyboard=True)
+    await message.answer(
+        _(
+            "Welcome to the CoverDanceBot control menu.\n\nClick add admin to add a user who can make"
+            "changes to the database.\n\nPress /help in the menu to see a description of all the actions"
```
Comment (CrispyCl) [inline bot/handlers/admin.py]: Add spaces to the text

File: bot/keyboards/set_menu.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
+from aiogram import Bot
+from aiogram.types import BotCommand
+
+
+async def set_main_menu(bot: Bot):
```
Comment (CrispyCl) [inline bot/keyboards/set_menu.py]: Rename to "setup_menu"

Comment (star-voyager-06) [reply bot/keyboards/set_menu.py]: Yeeeesss, I finally did it

File: bot/handlers/admin.py (hunk 1 above)
Comment (star-voyager-06) [reply bot/handlers/admin.py]: I rewrote this part

File: locales/ru/LC_MESSAGES/messages.po [hunk 3]
```diff
@@ -19,6 +19,232 @@ msgstr ""
...
+"<b>[🔍Find a practice video]</b> - Perform the video search procedure "
+"according to the specified parameters\n"
+"\n"
+"<b>[📽️Add video]</b> - Adding a new video to the bot's database\n"
+"\n"
+"<b>[🈹Change language]</b> - Changing the language of the bot interface"
+msgstr ""
+"<b>Раздел помощи админу</b>\n"
+"\n"
+"<b>[🔍Найти практик-видео]</b> - Выполнение поиска видео по по заданным "
```
Comment (CrispyCl) [inline locales/ru/LC_MESSAGES/messages.po]: "по по"

File: locales/en/LC_MESSAGES/messages.po [hunk 4]
```diff
@@ -18,6 +18,260 @@ msgstr ""
...
+
+#: bot/keyboards/admin.py:33
+msgid "❌Delete admin"
+msgstr ""
+
+#: bot/keyboards/admin.py:35 bot/keyboards/admin.py:43
+msgid "🔙Back"
+msgstr ""
+
+#~ msgid "Add admin"
```
Comment (CrispyCl) [inline locales/en/LC_MESSAGES/messages.po]: Delete the comments of old translations

File: locales/ru/LC_MESSAGES/messages.po [hunk 5]
```diff
@@ -19,6 +19,232 @@ msgstr ""
...
+msgid "This user is not an admin, try again or press [back]"
+msgstr ""
+"Этот пользователь не является админом, попробуйте ещё раз или нажмите "
+"[назад]"
+
+#: bot/handlers/admin.py:161
+msgid ""
+"<b>You cannot revoke your administrator rights</b>\n"
+"\n"
+"Enter another user name or press [back]"
```
Comment (CrispyCl) [inline locales/ru/LC_MESSAGES/messages.po]: username*

File: bot/middleware/logging.py [hunk 6]
```diff
@@ -28,6 +28,7 @@ async def __call__(
 
         except Exception as e:
             self.logger.error("<%d> %-7s: %s", update.update_id, "error", str(e))
+            raise e
```
Comment (CrispyCl) [inline bot/middleware/logging.py]: You're already logging the error, why raise it?

File: locales/ru/LC_MESSAGES/messages.po [hunk 7]
```diff
@@ -19,6 +19,232 @@ msgstr ""
...
+"<b>[📽️Add video]</b> - Adding a new video to the bot's database\n"
+"\n"
+"<b>[🈹Change language]</b> - Changing the language of the bot interface"
+msgstr ""
+"<b>Раздел помощи админу</b>\n"
+"\n"
+"<b>[🔍Найти практик-видео]</b> - Выполнение поиска видео по по заданным "
+"параметрам\n"
+"\n"
+"<b>[📽️Add video]</b> - Добавление нового видео в базу данных бота\n"
```
Comment (CrispyCl) [inline locales/ru/LC_MESSAGES/messages.po]: Translate the button

Comment (star-voyager-06) [reply locales/ru/LC_MESSAGES/messages.po]: Did it)

**Pull request merged**